wordpress-mobile/WordPress-iOS
# ULS: Show sized Prologue buttons

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/456, https://github.com/wordpress-mobile/WordPress-iOS/issues/14804
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/457

This uses the Auth changes to display the Prologue buttons sized on iPad.

To test:
- Run the app on an iPad.
- Verify:
  - In portrait, the buttons are 66% of the view width.
  - In landscape, the buttons are 50% of the view width.

Basically, verify they are the same width as the other unified views.

| Before | After |
|--------|-------|
| ![old_ipad_portrait](https://user-images.githubusercontent.com/1816888/92976207-42df3080-f447-11ea-9f93-f7f2418e71b0.png) | ![ipad_portrait](https://user-images.githubusercontent.com/1816888/92975689-0e1ea980-f446-11ea-903b-f14ab93e7c47.png) |
| ![old_ipad_landscape](https://user-images.githubusercontent.com/1816888/92976217-4bd00200-f447-11ea-8a87-3a103a4550ea.png) | ![ipad_landscape](https://user-images.githubusercontent.com/1816888/92975719-1f67b600-f446-11ea-994f-5b884a84d32e.png) |

PR submission checklist:
